test/certs/setup.sh

```diff
@@ -410,7 +410,7 @@ openssl req -new -noenc -subj "/CN=localhost" \
 ./mkcert.sh genct server.example embeddedSCTs1-key embeddedSCTs1 embeddedSCTs1_issuer-key embeddedSCTs1_issuer ct-server-key
 
 OPENSSL_SIGALG=ED448 OPENSSL_KEYALG=ed448 ./mkcert.sh genroot "Root Ed448" \
-    root-ed448-key root-ed448-cert
+    root-ed448-key root-ed448-cert # TODO updating this cert does not work
```

Comment:
Better than adding TODO to the code would be to create and issue.

Reply:
Hmm, actually it is just sufficient to replace `OPENSSL_SIGALG=ED448` with `OPENSSL_SIGALG=`

Reply:
Yep, would be nice if we can solve this right away.

With the trick you provided I now get a new cert (pushed as part of the fixup PR), but
```
apps/openssl x509 -noout -text -in test/certs/root-ed448-cert.pem
```
yields
```
Could not read certificate from test/certs/root-ed448-cert.pem
Unable to load certificate

Reply:
Weird, I do not see this issue with the current master. Also you seem to have produced empty file test/certs/root-ed448-cert.pem in the fixup.

Reply:
Argh, my `mkcert.sh` run had picked up an old openssl app version.
Now it works for me, too; pushed the resulting cert.